event.c

```diff
@@ -1926,6 +1942,8 @@ event_base_loop(struct event_base *base, int flags)
 	struct timeval tv;
 	struct timeval *tv_p;
 	int res, done, retval = 0;
+	struct event_watcher_cb_info watcher_cb_info;
```

Comment:
Do you want separate CB info for prepare/check?

Reply:
I didn't see an immediate need for that, but on second thought, the whole purpose of struct-ifying these is for future flexibility... So yes, yes I want that. Thanks!